cnf-certification-test/platform/ishyperthread/ishyperthread.go

```diff
@@ -0,0 +1,44 @@
+package ishyperthread
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/test-network-function/cnf-certification-test/internal/clientsholder"
+	"github.com/test-network-function/cnf-certification-test/pkg/provider"
+)
+
+const (
+	isHyperThreadCommand = "lscpu | grep \"Thread(s) per core\""
```

Comment:
use only lscpu and create a function similar to the extractNumber function to parse the output directly with regex
or
use lscpu --json and unmarshall the json output to get the number of threads per core